tests/test_endgame_tablebase.py

```diff
@@ -3,18 +3,20 @@
 from perf_helper import run_perf_analytics
 
 from sporkfish.board.board_factory import BoardPyChess
-from sporkfish.endgame_tablebase import EndgameTablebase, EndgameTablebaseConfig
+from sporkfish.endgame_tablebases.composite_tablebase import CompositeTablebase
+from sporkfish.endgame_tablebases.endgame_tablebase_config import EndgameTablebaseConfig
+from sporkfish.endgame_tablebases.lila_tablebase import LilaTablebase
 
 
 def move_from_et_query(fen: str):
     board = BoardPyChess()
     board.set_fen(fen)
-    et = EndgameTablebase(EndgameTablebaseConfig("data/endgame_tablebases"))
+    et = CompositeTablebase(EndgameTablebaseConfig("data/endgame_tablebases"))
```

Comment:
In principal I'd like to add another class TestCompositeTablebase, but let's leave it for now

Reply:
Fixed.